build/bin/kylin.sh

```diff
@@ -72,7 +72,14 @@ function retrieveDependency() {
     KYLIN_SPARK_JARS_HOME="${KYLIN_HOME}/spark/jars"
     if [[ -d "/usr/hdp/current/hdinsight-zookeeper" && $hdp_version == "2"* ]]
     then
-       echo "The current Hadoop environment is HDI3, will replace some jars package for ${KYLIN_HOME}/spark/jars"
+       echo "The current Hadoop environment is HDI3, will remove ${KYLIN_HOME}/tomcat/webapps/kylin/WEB-INF/lib/guava-14.0.jar and replace some jars package for ${KYLIN_HOME}/spark/jars"
+       unzip ${KYLIN_HOME}/tomcat/webapps/kylin.war
```

Comment:
It's better to check whether exists unzip command, if not, echo some error message and exit.

Reply:
My negligence. I wiil use `jar -xvf` to decompression kylin.war.